src/main/java/com/facebook/openwifirrm/optimizers/tpc/LocationBasedOptimalTPC.java

```diff
@@ -15,14 +15,14 @@
 import java.util.TreeMap;
 import java.util.stream.Collectors;
 
+import com.facebook.openwifirrm.ucentral.UCentralUtils;
```

Comment:
did you mean to move this? Also it seems like we separate our imports vs third-party

Reply:
No. No idea why it was moved. Maybe I did it by mistake.

Reply:
Seems like my Intellij IDEA automatically put the new import here. Checking my config now.